patches/MA-150---allow-update-browser-morphs.diff

```diff
@@ -0,0 +1,26 @@
+diff --git a/src/Repositories/Behaviors/HandleBrowsers.php b/src/Repositories/Behaviors/HandleBrowsers.php
+index c84fbbbf..4ffecd79 100644
+--- a/src/Repositories/Behaviors/HandleBrowsers.php
++++ b/src/Repositories/Behaviors/HandleBrowsers.php
```

Comment:
What is changed here?

Reply:
Lines 3 & 4 show if there is a change in the filename, which in this case there isn't.